pom.xml

```diff
@@ -228,33 +228,6 @@
             <version>2.13.2.1</version>
         </dependency>
 
-        <!-- Apache HTTP Components for actually sending requests over the network -->
```

Comment:
nit: may be do the refactoring of httpclient in separate PR to keep this only telemetry related. 

Also, thanks for doing it, no reason why we cant use it from jdbc!